api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -847,7 +75,7 @@ public void onFailure(Throwable t) {
     public void findMetrics(
             @Suspended final AsyncResponse asyncResponse,
             @PathParam("tenantId") final String tenantId,
-        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[num, avail, log]")
+        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[gauge, avail, log]")
```

Comment:
I'd prefer we simply get rid the query param option:
/{tenantId}/metrics -> returns all metrics, regardless of the type
/{tenantId}/gauges -> returns all gauges
/{tenantId}/counters -> returns all counters
/{tenantId}/avail -> returns all avail